### DISCUSSION TITLE
Synology @eaDir exclusion

### DISCUSSION THREAD

**Discussion by marco10x15** [bpatrik/pigallery2#604] [Q&A]
I have deplyed the docker on my synology NAS and pigallery2 is fantastic... but I was not able to exclude the "@eaDir" that Synology use for preview from to be indexed.

In the exclue directory I have:

.Trash-1000;.dtrash;$RECYCLE.BIN;#recycle;#recycle/;/#recycle;@eaDir;/@eaDir;.@eaDir;/.@eaDir;/.@eaDir/; @eaDir

then I restart  the indexing and the @eaDir are stil present.

What I'm doing wrong?

Thanks!

**Comment by Nionor** [comment]
When you change the exclude folder list it say something like reset database and restart server.
I had to do an extra reset database after the server restart to get it working. 

I only added the @eaDir at the end of the defaults and now it's gone everywhere.

Hope that helps!

**Reply by marco10x15** [reply]
Thanks, unfortunately fir me it is't working. how I reset the database?

**Reply by Nionor** [reply]
In settings click on 
![image](https://user-images.githubusercontent.com/36626165/218742046-e2a4af92-3113-41f3-8913-0f435d7f9634.png)

You might need to be in Mode: Advanced to see it.

**Comment by dima-ser** [comment]
I have the same issue. How were you able to resolve it, I tried the gallery reset, didn't work. Do you need to stop and restart the Docker container as well?

**Reply by Nionor** [reply]
It was a while ago now but yes I think I did.
Not sure if you can restart the server in another way.

I think the order how I did it was like this:
1. Change the exclude folder list
2. Save
3. Reset database
4. Restart container
5. Reset database

**Reply by bpatrik** [reply]
It's been a while when this got developed so my memories might be foggy:

The app only indexes the files when there is a change in the file system. It uses the excluding settings during this indexing. 
To test the exclude settings it should be enough to rename a file (or in it's parent) in a folder and navigate to that folder.

I think you only need to restart the server after the change because if you enable threading (enabled by default) indexing runs in a separate thread and the settings does not get synced to that thread.

Yes the only way to restart is to kill the container.
